pic-sure-resources/pic-sure-aggregate-data-sharing-resource/src/main/java/edu/harvard/hms/dbmi/avillach/ApplicationProperties.java

```diff
@@ -94,10 +94,13 @@ public void init(String contextPath) {
             throw new PicsureQueryException("target.picsure.token property must be set.");
         }
 
-        visualizationResourceId = UUID.fromString(properties.getProperty("visualization.resource.id"));
-        logger.debug("visualizationResourceId: " + visualizationResourceId);
-        if (visualizationResourceId == null)
-            throw new PicsureQueryException("visualization.resource.id property must be set.");
+	String visualizationResourceUUID = properties.getProperty("visualization.resource.id");
```

Comment:
I should call this AsString or something because the visualizationResourceId is an actual UUID type